### PR TITLE
Bump to the latest version of eslint-config-airbnb

### PR DESCRIPTION
**Pull request by yodaddyp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-bravissimo",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (lukebennett) [inline package.json]: Are there any backwards compatibility issues with the airbnb upgrade? I suspect this should be 2.1.0 or maybe even 3.0.0 - it's not a minor patch either way.

**Comment by yodaddyp** [comment]
@lukebennett Bumped major version change and removed some rules to ensure backwards compatibility.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+2.0.1 / 2017-03-10
```
Comment (lukebennett) [inline CHANGELOG.md]: Version number needs updating here.

Comment (yodaddyp) [reply CHANGELOG.md]: Good spot I have updated and pushed

**Review by lukebennett — APPROVED** [review]
LGTM

**Pull request merged**